quil-rs/src/instruction/snapshots/quil_rs__instruction__gate__test_gate_definition__display@Pauli Sum GateDefinition.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: quil-rs/src/instruction/gate.rs
+expression: gate_def.to_string()
+---
+DEFGATE PauliSumGate(%theta) AS PAULI-SUM:
```

Comment:
This is missing `p` and `q` in the signature

Reply:
Nice catch! Took a bit of refactoring, but their included now.